rclpy/rclpy/node.py

```diff
@@ -528,7 +536,7 @@ def _set_parameters(
         according to `raise_on_failure` flag.
 
         :param parameter_list: List of parameters to set.
-        :param descriptor_list: List of descriptors to set to the given parameters.
+        :param descriptors: Descriptors to set to the given parameters.
             If a list is given, it must have the same size as the parameter list.
```

Comment:
If a list is given ... -> If a dictionary is given, each parameter should have an entry in it.

Reply:
Thanks; addressed in https://github.com/ros2/rclpy/pull/357/commits/7381f62aeade16395c33dee943d7a5aa94173062.